### PR TITLE
feat: adds streaming samples

### PR DESCRIPTION
**Pull request by telpirion**
This pull request adds the following region tags / samples:
- media_translation_translate_from_file
- media_translation_translate_from_mic

Notes:
- The quickstart is a simple client instantiation. Streaming is a bit more involved than most quickstarts allow--if necessary, translate_from_file can be the quickstart.
- Not sure how to test the translate from microphone sample.

### CODE REVIEW

File: samples/quickstart.js [hunk 1]
```diff
@@ -1,38 +1,36 @@
-// Copyright 2020 Google LLC
-//
+// Copyright 2020, Google LLC.
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     https://www.apache.org/licenses/LICENSE-2.0
+//    http://www.apache.org/licenses/LICENSE-2.0
```
Comment (bcoe) [inline samples/quickstart.js]: this is a little weird, probably just leave it as `https://`, must be something we updated in our license at some point.

File: samples/quickstart.js [hunk 2]
```diff
@@ -1,38 +1,36 @@
...
-  const translate = new SpeechTranslationServiceClient();
-
-  // parent = 'projects/my-project', // Project to list dashboards for.
-
-  // TODO: add an actual sample.
-  console.info(translate);
-
+  async function quickstart() {
+    const client = new SpeechTranslationServiceClient();
+    console.log(client);
```
Comment (bcoe) [inline samples/quickstart.js]: we're not actually doing anything async in this sample yet, is there a minimal example we could toss in here?

File: samples/translate_from_file.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+      highWaterMark: 4096,
+      encoding: 'base64',
+    });
+
+    const chunks = [];
+    readStream
+      .on('data', chunk => {
+        const request = {
+          streamingConfig: config,
+          audioContent: chunk.toString(),
```
Comment (bcoe) [inline samples/translate_from_file.js]: I can't wait for [async iterators](https://github.com/tc39/proposal-async-iteration), which land in Node 10, I believe they'll let us clean up this code a bunch eventually.

This is good for now, just putting this new approach on your radar.

File: samples/quickstart.js (hunk 2 above)
Comment (telpirion) [reply samples/quickstart.js]: Good point. I copied the translate_from_file.js into the quickstart.

File: samples/translate_from_file.js (hunk 3 above)
Comment (telpirion) [reply samples/translate_from_file.js]: Oh, that would have been so helpful! Can't wait to learn more.

File: samples/quickstart.js (hunk 1 above)
Comment (telpirion) [reply samples/quickstart.js]: Updated.

**Pull request merged**